emcc.py

```diff
@@ -3555,9 +3555,15 @@ def minify_html(filename, options):
     return
 
   logger.debug('minifying HTML file ' + filename)
+  env = os.environ.copy()
+  env['PATH'] = env['PATH'] + os.pathsep + shared.get_node_directory()
```

Comment:
I wonder if we should do this in central location since we do it for closure too.